runners/dmn-tck-runner-drools/src/test/java/org/omg/dmn/tck/runner/drools/DroolsTCKTest.java

```diff
@@ -57,9 +71,49 @@
    implements DmnTckVendorTestSuite
 {
 
+   private static final SimpleFileVisitor<Path> rmrf = new SimpleFileVisitor<Path>() {
+      @Override
+      public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
+         Files.delete(file);
+         return FileVisitResult.CONTINUE;
+      }
+
+      @Override
+      public FileVisitResult postVisitDirectory(Path dir, IOException exc) throws IOException {
+         Files.delete(dir);
+         return FileVisitResult.CONTINUE;
+      }
+   };
    private static final Logger logger = LoggerFactory.getLogger(DroolsTCKTest.class);
    public static final BigDecimal NUMBER_COMPARISON_PRECISION = new BigDecimal("0.00000001");
 
+   File propsFile;
+   Properties props;
+
+   {
+      File dir = new File("../../TestResults/Drools");
+      String[] versions = dir.list((f, s) -> f.isDirectory());
+      assert versions != null;
```

Comment:
I don't think we can rely on `assert` as there is no requirement to run the JVM with the `-ea` flag. Am I missing something?

Reply:
It will fail later anyway. I am writing asserts more as a best practice and means of documentation, pointing out presence of versions directories here is mandatory to proceed.

Reply:
The problem is that instead of handling the assert-fail case gracefully, it is crashing as per other comment, applicable here.